package-lock.json

```diff
@@ -9300,37 +9327,59 @@
       }
     },
     "regl-scatter2d": {
-      "version": "3.1.4",
-      "resolved": "https://registry.npmjs.org/regl-scatter2d/-/regl-scatter2d-3.1.4.tgz",
-      "integrity": "sha512-X3FNkh3lw4B3cGLpGZYhSXoeeNAYYeAZnjoP4R8+uCmCbBiwoXUiLQw73HxjsCs53l/upzrSLCwu13QZy/MHeQ==",
+      "version": "git://github.com/gl-vis/regl-scatter2d.git#09889cb65fc6b662b7f184ef7da8c3c87c685ad5",
+      "from": "git://github.com/gl-vis/regl-scatter2d.git#09889cb65fc6b662b7f184ef7da8c3c87c685ad5",
       "requires": {
         "array-range": "^1.0.1",
         "array-rearrange": "^2.2.2",
         "clamp": "^1.0.1",
         "color-id": "^1.1.0",
-        "color-normalize": "^1.3.0",
-        "color-rgba": "^2.1.0",
+        "color-normalize": "git://github.com/archmoj/color-normalize.git#24af8e56998e2013f49baca2f8e3338c6bf2ce6c",
+        "color-rgba": "git://github.com/archmoj/color-rgba.git#92f2babae20c1eec60fd9185719baf3f85a7ba56",
         "flatten-vertex-data": "^1.0.2",
         "glslify": "^7.0.0",
         "image-palette": "^2.1.0",
         "is-iexplorer": "^1.0.0",
         "object-assign": "^4.1.1",
         "parse-rect": "^1.2.0",
         "pick-by-alias": "^1.2.0",
-        "point-cluster": "^3.1.4",
+        "point-cluster": "^3.1.5",
```

Comment:
Hmm. Do we really want to bump `point-cluster`?

Reply:
It has already been applied within the master branch of `regl-scatter2d`: https://github.com/gl-vis/regl-scatter2d/commit/cda45ae30e5e1f841d65ced2404c9e36a618415f. Yet we don't have a new reg-scatter2d release.

Reply:
Right, so can we wait until we need to bump `regl-scatter2d` to bump `point-cluster`?

Reply:
Yes. Exactly.

Reply:
... unless you feel comfortable with https://github.com/dy/point-cluster/commit/8419f968f7818ce9f347e7ac2bf4940f2fcfa337

Reply:
Ah I see, `regl-scatter2d` uses `color-parse`. Yep this package-lock diff makes perfect sense. Sorry for the confusion :ok_hand: